nautobot/extras/migrations/0055_collect_roles_from_related_apps_roles.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.2.16 on 2022-11-19 23:15
+
+from collections import namedtuple
+from django.db import migrations, models
+
+from nautobot.ipam.choices import IPAddressRoleChoices
```

Comment:
NOTE: This would be removed when removing obsolete code; either hardcode '`IPAddressRoleChoices`' choices here or do not remove '`IPAddressRoleChoices`'. Because its been used in data migration below,